src/Stratis.Features.FederatedPeg/Collateral/CheckCollateralFullValidationRule.cs

```diff
@@ -66,6 +66,8 @@ public override Task RunAsync(RuleContext context)
             int commitmentHeight = this.encoder.Decode(rawCommitmentData);
             this.Logger.LogDebug("Commitment is: {0}.", commitmentHeight);
 
+            // TODO: Both this and CollateralPoAMiner are using this chain's MaxReorg instead of the Counter chain's MaxReorg. Beware: fixing requires fork.
```

Comment:
Obviously not related to this PR, just wanted to slip it in somewhere so that "future us" is aware